app/assets/stylesheets/common/base/sidebar.scss

```diff
@@ -138,6 +138,27 @@
     margin-top: 0.15em;
     align-items: center;
   }
+
+  .sidebar-section-link-prefix.icon {
```

Comment:
Each "component" in sidebar has its own CSS file so I think we can try to move these into its respective files instead.